src/shogun/transfer/multitask/MultitaskKernelPlifNormalizer.h

```diff
@@ -328,10 +328,22 @@ class CMultitaskKernelPlifNormalizer: public CMultitaskKernelMklNormalizer
 	{
 		SG_ADD(&num_tasks, "num_tasks", "the number of tasks", MS_NOT_AVAILABLE);
 		SG_ADD(&num_betas, "num_betas", "the number of weights", MS_NOT_AVAILABLE);
+
 		m_parameters->add_vector((SGString<float64_t>**)&distance_matrix, &num_tasksqr, "distance_matrix", "distance between tasks");
+		watch_param(
+			"distance_matrix", (SGString<float64_t>**)&distance_matrix,
```

Comment:
I don't think this can work, these static casts don't make any sense
Could you remove you added watch_param statements here?

Then, in another PR, could you even comment out the add_vector calls?

Reply:
Ok.